spec/formatting.md

```diff
@@ -474,6 +474,11 @@ This selection method is defined in more detail below.
 An implementation MAY use any pattern selection method,
 as long as its observable behavior matches the results of the method defined here.
 
+The resolved value of each _key_ MUST be in Unicode Normalization Form C ("NFC"),
+even if the _literal_ for the _key_ is not.
+All comparisons of _keys_ MUST be done on the canonical, normalized values
+and the normalized value MUST be the value that is passed in the steps below.
```

Comment:
This should probably be further below, after the paragraph starting with "The method MatchSelectorKeys is determined ...".

It's also very MUST...MUST...MUST. Is that really necessary? Does the second sentence really add anything beyond the first?

Reply:
Agreed. 

There are the resolved values passed to MatchSelectorKeys, but then there is the matching done checking for duplicates. I could leave out the third MUST... perhaps:

```suggestion
The resolved value of each _key_ MUST be in Unicode Normalization Form C ("NFC"),
even if the _literal_ for the _key_ is not, 
and all comparisons of _keys_ MUST be done on this canonical, normalized form.
```

Reply:
Better. Do note that the check for duplicates is a syntax/data model matter; this is about pattern selection. Any MUST about the former ought to be (and is already) under Syntax; it should not be here.

Reply:
Done